Makefile

```diff
@@ -352,8 +352,7 @@ case.h
 
 compile: \
 make-compile warn-auto.sh
-	( cat warn-auto.sh; ./make-compile ) > \
```

Comment:
unrelated change

Reply:
It's one of a couple inconsistencies I noticed while preparing `instqueue`. They're in separate commits that seemed small enough and related enough to include here. If we prefer them in a distinct PR or PRs, okay.

Reply:
Ok, it's intentional, let's just do it. Noone is going to modify that line in a patch anyway.